mermin/src/metrics/opts.rs

```diff
@@ -18,6 +20,7 @@ impl Default for MetricsOptions {
             enabled: true,
             listen_address: Ipv4Addr::UNSPECIFIED.to_string(),
             port: 10250,
+            debug_enabled: true,
```

Comment:
Since log_level defaults to `info`, I feel that `debug_enabled` should default to `false`. The default runtime should be assumed to not be in "debug" mode.